doc/news/changes.h

```diff
@@ -428,6 +428,13 @@ SparsityPattern.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> New: PreconditionMG can now be used as a LinearOperator. Implement
+ MGCoarseGridApplySmoother class to do a few steps of a smoother at
+ the coarsest level.
```

Comment:
I suggest you split this into two list items since the coarse grid smoother and LinearOperator parts are unrelated.